linhpsdr.desktop

```diff
@@ -1,9 +1,7 @@
 [Desktop Entry]
-Version=0.1
```

Comment:
This is version of the desktop file specification, not the program version. The 0.1 is invalid desktop file specificaiton version. This entry is optional, thus drop it.